tests/qnodes/test_qnode_decorator.py

```diff
@@ -97,6 +98,25 @@ def circuit(a):
     assert isinstance(circuit, JacobianQNode)
     assert hasattr(circuit, "jacobian")
     assert circuit.h == step_size
+    assert circuit.order == 1
+
+
+@pytest.mark.parametrize("order", [1, 2])
+def test_finite_diff_qubit_qnode_passing_order_through_decorator(order):
+    """Test that a finite-difference differentiable qubit QNode is correctly created when
+    diff_method='finite-diff' and the order is set through the decorator."""
+    dev = qml.device('default.qubit', wires=1)
+
+    @qnode(dev, diff_method="finite-diff", order=order)
+    def circuit(a):
+        qml.RX(a, wires=0)
+        return qml.expval(qml.PauliZ(wires=0))
+
+    assert circuit.order == order
+
+    circuit.order = 1
+    assert circuit.order == 1
```

Comment:
It might also be good to write a unit test to make sure that the options dictionary passed to `_pd_finite_diff` correctly includes `"order":2` (since these tests are mostly just testing the setter/property).

Reply:
Great, added: 63e584d.